src/components/matching/MatchingUserInfo.tsx

```diff
@@ -1,24 +1,36 @@
 import styled from '@emotion/styled';
 import Image from 'next/image';
 
+import { useRouter } from 'next/router';
+import { DEFAULT_PROFILE_IMAGES } from 'constants/profile';
 import { useTimer } from 'hooks/common/useTimer';
+import { useProfile } from 'hooks/services';
 import { ScreenReaderOnly } from 'styles';
 
 const MatchingUserInfo = () => {
   const { minutes, seconds } = useTimer();
 
+  const router = useRouter();
+
+  const { query } = router;
+
+  const { profileData: matchingUserProfile } = useProfile(
```

Comment:
p3;

다른 프로필 데이터가 추가되는 것이 아니라면 변수명 수정 없이 바로 사용하시는 것이 좋을 것 같아요!
```suggestion
  const { profileData } = useProfile(
```

Reply:
좋은 의견 감사합니다! :)

profileData라는 변수명이 본인의 프로필 데이터인지, 아니면 매칭된 유저의 프로필 데이터인지 헷갈릴 것으로 생각되어 명시적으로 위와 같은 변수명을 사용하게 되었습니다!